### PR TITLE
Add Bengali langauage

### PR DESCRIPTION
**Pull request by wasi-master**
Adds the language called [Bengali](https://en.wikipedia.org/wiki/Bengali_language) which is spoken by [Bengalis](https://en.wikipedia.org/wiki/Bengalis) and [60 others according to google](https://www.google.com/search?q=bengali+language+spoken+by) that mostly live in [Bangladesh](https://en.wikipedia.org/wiki/Bangladesh) and [Kolkata](https://en.wikipedia.org/wiki/Kolkata)

The language is spoken by approximately 228 million native speakers and another 37 million as second-language speakers and [ranks 7th](https://www.visualcapitalist.com/100-most-spoken-languages) among the most spoken languages in the world

### CODE REVIEW

File: src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-07-29 21:59+0600\n"
+"PO-Revision-Date: 2021-07-29 22:07+0600\n"
+"Last-Translator: U-WASI-PC\\Wasi Master <arianmollik323@gmail.com>\n"
+"Language-Team: Bengali\n"
+"Language: bn_BD\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
```
Comment (hugovk) [inline src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po:16]: We'll need UTF-8 here: 
```suggestion
"Content-Type: text/plain; charset=UTF-8\n"
```

File: src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```
Comment (hugovk) [inline src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po:18]: And we'll need the correct values in here.

**Review by hugovk — CHANGES_REQUESTED** [review]
Thanks for the PR! Please can you update these couple of things, and add the language to the README list? https://github.com/jmoiron/humanize#humanize

File: src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po (hunk 2 above)
Comment (hugovk) [reply src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po:18]: Please replace `INTEGER` and `EXPRESSION`

See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html which has examples for other language families.

File: src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po [hunk 3]
```diff
@@ -0,0 +1,291 @@
+# Bengali translations for humanize package.
+# Copyright (C) 2021 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the humanize package.
+# Wasi Master <arianmollik323@gmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```
Comment (hugovk) [inline src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po:8]: This is the name and version of the package.

You can do something like one of the other language files: 

```
src/humanize/locale/ca_ES/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/de_DE/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/es_ES/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/fa_IR/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/fi_FI/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/fr_FR/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/id_ID/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/it_IT/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/ja_JP/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/ko_KR/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/nl_NL/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/pl_PL/LC_MESSAGES/humanize.po:"Project-Id-Version: 0.0.1\n"
src/humanize/locale/pt_BR/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/pt_PT/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/sk_SK/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/sl_SI/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/sv_SE/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/tr_TR/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/uk_UA/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/vi_VN/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/zh_CN/LC_MESSAGES/humanize.po:"Project-Id-Version: 1.0\n"
```

**Comment by wasi-master** [comment]
okay I resolved all the issues

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
Thank you, released in 3.11.0!

**Comment by wasi-master** [comment]
:)

**Comment by wasi-master** [comment]
Wait I forgot to add the language to readme.md

**Comment by hugovk** [comment]
No problem, I'll do it!

**Pull request merged**